use-timescale/data-tiering/disabling-data-tiering.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Creating a tiering policy
+excerpt: How to create a tiering policy
+product: [ cloud ]
+keywords: [ tiered storage, tiering ]
+tags: [ storage, data management ]
+---
+
+# Disabling tiering on a hypertable
+
+If you no longer want to use tiered storage for a particular hypertable, you 
+can drop the associated metadata by calling the `disable_tiering` function.
```

Comment:
Since this may not be obvious to a user, maybe it makes sense to clarify, that untiering/dropping all the tiered chunks does not automatically disable tiering. And since it can affect query planning it may be beneficial to disable it.

Reply:
This is explained later on.